core/src/modules/ops/property.ts

```diff
@@ -14,22 +16,34 @@ export namespace PropertyOps {
    */
   export async function enqueueRuns(property: Property) {
     if (config.general.runMode === "cli:validate") return;
+    if (config.general.runMode === "cli:config") {
+      await RecordProperty.update(
+        { state: "pending", startedAt: null },
+        { where: { propertyId: property.id } }
+      );
+
+      const source = await property.$get("source", { scope: null });
+      await GrouparooRecord.update(
+        { state: "pending" },
+        { where: { modelId: source.modelId } }
+      );
```

Comment:
In `cli:confg` mode, we'll do the pending updates in-line.  We don't need to worry about imports or groups in this mode.